client/js/views/MyProfile.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <v-container class="profile">
+    <h1>{{ user }} Profile Settings</h1>
+    <br />
+    <v-text-field
+      v-model="email"
+      outlined
+      :dense="$vuetify.breakpoint.mdAndDown ? true : false"
+      hint="email"
+      data-cy="email"
+      autocomplete="email"
+      placeholder="email"
+      @keydown.enter.prevent="submitEmail"
+    />
+    <v-snackbar
+      v-model="showSnackBar"
+      :color="colorValue"
+      content-class="d-flex justify-space-between align-center"
+      data-cy="auth-snackbar"
+    >
+      {{ snackBarMessage }}
+      <v-icon data-cy="close-snackbar" @click="clearSnackBar"> mdi-close </v-icon>
+    </v-snackbar>
+    <!-- <h3>Current Email: test</h3> -->
+  </v-container>
+</template>
+<script>
+export default {
+  name: 'MyProfile',
+  data() {
+    return {
+      user: this.$store.state.auth.username,
+      email: '',
+      showSnackBar: false,
+      snackBarMessage: '',
+      colorValue: '',
+    };
+  },
+  computed: {},
+  mounted() {},
+  methods: {
+    async submitEmail() {
+      try {
+        //Email Testing Regex
+        if (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(this.email)) {
+          //ToDo: Figure how to save email to Database
```

Comment:
Great start. We'll need this to work full stack before we merge&deploy so we don't ship a broken feature.

The gist is we need a new backend endpoint, defined in `api/controllers/UserController.js` which we could call submitEmail() that would expect that the user sends an email in the request body and if so it would call User.updateOne() to update the correct user and add their email. We can discuss the details outside GitHub but the other key pieces will be 

adding a backend route in config/routes.js to map the new endpoint to

adding a new store action in the auth store module that fires a request to this new endpoint and returns the result as a promise. That'll let this page dispatch the store action to make the request and handle success/error based on that request